stargate-starter/src/main/java/io/stargate/starter/Starter.java

```diff
@@ -426,6 +431,11 @@ public void start() throws BundleException {
       bundle.start();
     }
 
+    if (startError.get()) {
+      System.out.println("Terminating due to previous service startup errors.");
```

Comment:
maybe log the `startError` here as well?
```
 System.out.println("Terminating due to previous service startup errors: ", startError.get());
```

Reply:
It's just a `boolean` and always `true` at this point. Is it really worth logging?

Reply:
Root cause errors/exceptions are logged by OSGi in this case.

Reply:
We just need to make the java process terminate because OSGi will not automatically terminate even if some serivices (e.g. persistence) fail to start.